docs/programming_guide/controllers/cross_site_model_evaluation.rst

```diff
@@ -2,14 +2,23 @@
 
 Cross Site Model Evaluation / Federated Evaluation
 --------------------------------------------------
-The cross site model evaluation workflow uses the data from clients to run evaluation with the models of other clients.
+The :class:`cross site model evaluation workflow<nvflare.app_common.workflows.cross_site_model_eval.CrossSiteModelEval>`
+uses the data from clients to run evaluation with the models of other clients.
 Data is not shared, rather the collection of models is distributed to each client site to run local validation.  The
 results of local validation are collected by the server to construct an all-to-all matrix of model performance vs.
 client dataset.
 
 The server’s global model is also distributed to each client for evaluation on the client’s local dataset for global
 model evaluation.
 
-See the code for details:
+The `hello-numpy-cross-val example <https://github.com/NVIDIA/NVFlare/tree/main/examples/hello-pt-tb>`_ is a simple
```

Comment:
should this link be "https://github.com/NVIDIA/NVFlare/tree/main/examples/hello-numpy-cross-val"?

Reply:
yes, thank you for spotting this